pkg/roachprod/vm/gce/utils.go

```diff
@@ -221,6 +221,13 @@ echo "kernel.core_pattern=$CORE_PATTERN" >> /etc/sysctl.conf
 sysctl --system  # reload sysctl settings
 
 sudo touch /mnt/data1/.roachprod-initialized
+{{ if .EnableFIPS }}
+sudo ua enable fips --assume-yes
+if [[ $(cat /proc/sys/crypto/fips_enabled) != "1" ]]; then
+  echo "FIPS mode is not enabled, exiting..."
+  exit 1
+fi
+{{ end }}
```

Comment:
Can we move this up? The `sudo touch /mnt/data1/.roachprod-initialized` line has to be the last one, as it tells roachprod that all setup commands are finished.

Reply:
I moved it and removed the `if` part, because it doesn't actually halt the tests.